Add commands to view DAG (#764)
flyteorg/flytesnacks: cookbook/docs/userguide_setup.rst

.. prompt:: bash

    pyflyte run --remote core/flyte_basics/basic_workflow.py my_wf --a 5 --b hello
    
.. note::

    We're passing in the workflow inputs as additional options to ``pyflyte run``, in the above example the
    inputs are ``--a 5`` and ``--b hello``. For snake-case argument names like ``arg_name``, you can provide the
    option as ``--arg-name``.

> Visualize the workflow

Reviewer: Visualizing Workflows

> Visualize the workflow

Reviewer: Visualizing Workflows

"""""""""""""""""""""""

> You can visualize the workflow without using the UI and the DAG can be rendered in the web browser.

Reviewer: ```suggestion
Workflows can be visualized as DAGs on the UI. However, if UI isn't your cup of tea, you can visualize workflows on the browser and in the terminal by *just* using your terminal.
```

> You can visualize the workflow without using the UI and the DAG can be rendered in the web browser.

Reviewer: ```suggestion
Workflows can be visualized as DAGs on the UI. However, if UI isn't your cup of tea, you can visualize workflows on the browser and in the terminal by *just* using your terminal.
```

> 

Reviewer: To view workflow structure on the browser:

> 

Reviewer: To view workflow structure on the browser:

.. prompt:: bash $

    flytectl get workflows --project flytesnacks --domain development flyte.workflows.example.my_wf --version v1 -o doturl

> You can also view the workflow as a ``strict digraph`` on the command line.

Reviewer: ```suggestion
To view workflow as a ``strict digraph`` on the command line:
```

> You can also view the workflow as a ``strict digraph`` on the command line.

Reviewer: ```suggestion
To view workflow as a ``strict digraph`` on the command line:
```

.. prompt:: bash

    pyflyte run --remote core/flyte_basics/basic_workflow.py my_wf --a 5 --b hello
    
.. note::

    We're passing in the workflow inputs as additional options to ``pyflyte run``, in the above example the
    inputs are ``--a 5`` and ``--b hello``. For snake-case argument names like ``arg_name``, you can provide the
    option as ``--arg-name``.

Visualizing Workflows
^^^^^^^^^^^^^^^^^^^^^

> Workflows can be visualized as DAGs on the UI. You can visualize workflows on the browser and in the terminal by *just* using your terminal.

Reviewer: ```suggestion
Workflows can be visualized as DAGs on the UI. However, you can visualize workflows on the browser and in the terminal by *just* using your terminal.
```

> To view workflow on the browser:

Reviewer: ```suggestion
To view a workflow on the browser:
```
